CONTRIBUTING.md

```diff
@@ -153,10 +153,10 @@ If you want to add a new npm package to 'node_modules' you will need to `--save-
 2. Create a synced npm-shrinkwrap.json with devDependencies included by running
 
     ```
-    npm run shrinkwrap
+    npm run build-shrinkwrap
```

Comment:
Why so verbose? `build`

Reply:
because it seemed like just shrinkwrap caused an infinite loop. It also gets executed when npm shrinkwrap is called, therefore calling itself over and over again

Reply:
Makes sense, thanks for the explanation

Reply:
👍